src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -46,7 +46,8 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
     userAuthedRequest(Get(RawlsDAO.groupUrl(groupName)))(userInfo) map { response =>
       response.status match {
         case OK => true
-        case _ => false
+        case Forbidden => false
```

Comment:
Per the docs on the Rawls endpoint `GET /api/user/group/{groupName}`, it actually returns `404` instead of `403` for `Group not found or user is not a member`

Might want to double-check that the implementation matches the Swagger